src/App.js

```diff
@@ -7,8 +7,14 @@ class App extends Component {
     return (
       <div className="App">
         <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
```

Comment:
Always follow consistent naming conventions for css selectors, functions, variables, files and folders.